vajram/vajram-codegen/src/main/java/com/flipkart/krystal/vajram/codegen/VajramCodeGenerator.java

```diff
@@ -203,7 +203,7 @@ public String codeGenVajramImpl(ClassLoader classLoader) {
       final TypeSpec.Builder vajramImplClass = createVajramImplClass();
       List<MethodSpec> methodSpecs = new ArrayList<>();
       // Add superclass
-      vajramImplClass.addModifiers(PUBLIC)
+      vajramImplClass.addModifiers(PUBLIC).addModifiers(FINAL)
```

Comment:
add modifiers method takes a varargs so can change to "addModifiers(PUBLIC, FINAL)".

Reply:
Oh super.